app/editor/src/features/content/form/components/time-log/interfaces/ITimeTrackingForm.ts

```diff
@@ -0,0 +1,5 @@
+import { ITimeTrackingModel } from 'tno-core';
+
+export interface ITimeTrackingForm extends ITimeTrackingModel {
```

Comment:
Always create a separate interface to capture additional information for the form.